breeze

```diff
@@ -1862,6 +1864,7 @@ ${FORMATTED_STATIC_CHECKS}
       '${CMDNAME} static-check mypy' or
       '${CMDNAME} static-check mypy -- --files tests/core.py'
       '${CMDNAME} static-check mypy -- --all-files'
+      '${CMDNAME} static-check all -- --from-ref `git merge-base master HEAD` --to-ref HEAD # checks all files that differ from master'
```

Comment:
The  ` wil have to be escaped - otherwise, it will try to run the git commnd.

Reply:
I changed it to `\$(git merge-base... )` and move the comment at the end as new line.